firmware/library/L2_HAL/memory/sddriver_test.cpp

```diff
@@ -0,0 +1,15 @@
+// Tests for the SD Card Driver Class
+#include "L3_HAL/sddriver.hpp"
+#include "L5_Testing/testing_frameworks.hpp"
+
```

Comment:
```suggestion
EMIT_ALL_METHODS(Sd);
```

This will cause your class to show up in the coverage evaluation. This should lower our code coverage, but that is fine for now.